src/utils/AttachVariableChangeListener.js

```diff
@@ -1,3 +1,8 @@
+/**
+ *
+ * @param {string} title
+ * Title of the variable
+ */
```

Comment:
Attaches listener to content controls so that similar instances of variables are updated simultaneously.